shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -73,6 +73,29 @@ FLUTTER_DARWIN_EXPORT
  */
 - (void)onPreEngineRestart;
 
+/**
+ * Returns the file name for the given asset.
+ * The returned file name can be used to access the asset in the application's
+ * main bundle.
+ *
+ * @param asset The name of the asset. The name can be hierarchical.
+ * @return The file name to be used for lookup in the main bundle.
+ */
+- (nonnull NSString*)lookupKeyForAsset:(nonnull NSString*)asset;
+
+/**
+ * Returns the file name for the given asset which originates from the specified
+ * package.
+ * The returned file name can be used to access the asset in the application's
+ * main bundle.
+ *
+ * @param asset The name of the asset. The name can be hierarchical.
+ * @param package The name of the package from which the asset originates.
+ * @return The file name to be used for lookup in the main bundle.
+ */
+- (nonnull NSString*)lookupKeyForAsset:(nonnull NSString*)asset
```

Comment:
Is there a reason these APIs are on FVC instead of just being public on the engine, since `engine` is public, and they are just passthroughs to the engine?

Reply:
This was marked as resolved with no comment or change.

Reply:
Just keep the same with iOS, and it's not pass through to the engine, the implementation belongs to `FlutterPluginRegistrar` in Flutter Engine source file.